src/chrome/content/rules/mvd.ru.xml

```diff
@@ -1,675 +1,12 @@
-	<target host="62.mvd.ru" />
-	<target host="63.mvd.ru" />
-	<target host="64.mvd.ru" />
-	<target host="65.mvd.ru" />
-	<target host="66.mvd.ru" />
-	<target host="67.mvd.ru" />
-	<target host="68.mvd.ru" />
-	<target host="69.mvd.ru" />
-	<target host="70.mvd.ru" />
-	<target host="71.mvd.ru" />
-	<target host="72.mvd.ru" />
-	<target host="73.mvd.ru" />
-	<target host="74.mvd.ru" />
-	<target host="75.mvd.ru" />
-	<target host="76.mvd.ru" />
-	<target host="77.mvd.ru" />
-	<target host="78.mvd.ru" />
-	<target host="79.mvd.ru" />
-	<target host="82.mvd.ru" />
-	<target host="83.mvd.ru" />
-	<target host="86.mvd.ru" />
-	<target host="87.mvd.ru" />
-	<target host="89.mvd.ru" />
-	<target host="92.mvd.ru" />
-	<target host="95.mvd.ru" />
-	<target host="a.mvd.ru" />
-	<target host="asvu.mvd.ru" />
-	<target host="belui.mvd.ru" />
-	<target host="bui.mvd.ru" />
-	<target host="cfout.mvd.ru" />
-	<target host="chsvu.mvd.ru" />
-	<target host="csr.mvd.ru" />
-	<target host="dfout.mvd.ru" />
-	<target host="dvui.mvd.ru" />
-	<target host="en.mvd.ru" />
-	<target host="ershsrs.mvd.ru" />
-	<target host="esiirk.mvd.ru" />
-	<target host="gcsp.mvd.ru" />
-	<target host="gibdd.mvd.ru" />
-	<target host="gsvu.mvd.ru" />
-	<target host="gut.mvd.ru" />
-	<target host="guvm.mvd.ru" />
-	<target host="en.guvm.mvd.ru" />
-	<target host="limited.guvm.mvd.ru" />
-	<target host="pda.guvm.mvd.ru" />
-	<target host="gvm.mvd.ru" />
-	<target host="krdu.mvd.ru" />
-	<target host="krmut.mvd.ru" />
-	<target host="kui.mvd.ru" />
-	<target host="mail.mvd.ru" />
-	<target host="media.mvd.ru" />
-	<target host="mosu.mvd.ru" />
-	<target host="na.mvd.ru" />
-	<target host="ncbdd.mvd.ru" />
-	<target host="noreply.mvd.ru" />
-	<target host="nsvu.mvd.ru" />
-	<target host="oma.mvd.ru" />
-	<target host="orl.mvd.ru" />
-	<target host="orui.mvd.ru" />
-	<target host="os.mvd.ru" />
-	<target host="pfout.mvd.ru" />
-	<target host="radio.mvd.ru" />
-	<target host="rshsrs.mvd.ru" />
-	<target host="rui.mvd.ru" />
-	<target host="sibfout.mvd.ru" />
-	<target host="skfo.mvd.ru" />
-	<target host="skfout.mvd.ru" />
-	<target host="skk.mvd.ru" />
-	<target host="static.mvd.ru" />
-	<target host="svuspb.mvd.ru" />
-	<target host="szfout.mvd.ru" />
-	<target host="tipk.mvd.ru" />
-	<target host="uc.mvd.ru" />
-	<target host="ufali.mvd.ru" />
-	<target host="ufout.mvd.ru" />
-	<target host="umvd72.mvd.ru" />
-	<target host="univer.mvd.ru" />
-	<target host="ural.mvd.ru" />
-	<target host="uralfout.mvd.ru" />
-	<target host="ushpsk.mvd.ru" />
-	<target host="va.mvd.ru" />
-	<target host="veteran.mvd.ru" />
-	<target host="vi.mvd.ru" />
-	<target host="vipk.mvd.ru" />
-	<target host="vnii.mvd.ru" />
-	<target host="vsut.mvd.ru" />
-	<target host="zbut.mvd.ru" />
+	<target host="*.mvd.ru" />
+		<test url="http://www.mvd.ru/" />
+		<test url="http://static.mvd.ru/upload/site1/05IWFr3NZU.png" />
+		<test url="http://static.mvd.ru/media/mvd-2015/css/flexlist.css" />
```

Comment:
So from >500 testable targets we end up with two hosts...😐
Furthermore, some of the certs in the comments are currently still expired or mismatch.